040-SDK/160-image-transformations.mdx

```diff
@@ -320,6 +320,38 @@ https://us-east-1.storage.xata.sh/transform/contrast=0.5/4u1fh2o6p10blbutjnphcst
 
 <TransformImageExample url="https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94" options={{ contrast: 0.5 }} />
 
+### download
+
+Option to force browser to [download](https://us-east-1.storage.xata.sh/transform/download=xata-cat.jpg,format=jpeg/4u1fh2o6p10blbutjnphcste94) the image instead of displaying it. Value is the file name and extension is optional.
```

Comment:
```suggestion
Depending on the file type, some browsers preview the file instead of downloading it. Setting the `download` property will make the browser to always download the file. The value allows you to assign a name, and optionally an extension to the file as it downloads.
```

Reply:
Looks better :+1: